### PR TITLE
correct the OADP sts install docs

### PR DESCRIPTION
**Pull request by weshayutin**
* use oadp 1.2.0 vs. 1.0
* correct malformed DPA
* ensure datamover is not enabled
* Add DPA for CSI volumes and non-CSI Volumes
* snapshotlocation is verified for OADP + ROSA STS
* The following roles were added for non-csi volume snapshot ( thanks tiger! )
         "ec2:DescribeVolumes",
         "ec2:DescribeVolumeAttribute",
         "ec2:DescribeVolumesModifications",
         "ec2:DescribeVolumeStatus",

### CODE REVIEW

File: content/docs/misc/oadp/rosa-sts/_index.md [hunk 1]
```diff
@@ -226,13 +228,6 @@ and restore process, but it should be noted as there are issues with it.
          - aws
        restic:
          enable: false
-     volumeSnapshots:
```
Comment (shawn-hurley) [inline content/docs/misc/oadp/rosa-sts/_index.md:229]: We might need this right to ensure the enableSharedConfig is true, or am I wrong?

**Review by shawn-hurley — COMMENTED** [review]
Only worried about the volumeSnapshots but otherwise I think makes sense.

VISACK

File: content/docs/misc/oadp/rosa-sts/_index.md (hunk 1 above)
Comment (kaovilai) [reply content/docs/misc/oadp/rosa-sts/_index.md:229]: If someone can demo to wes VSL works in this fashion he'd add it back in later.

Comment (kaovilai) [reply content/docs/misc/oadp/rosa-sts/_index.md:229]: I don't think @weshayutin followed mobb docs exactly tho.. he didn't have config set like this.
```
     - velero:
         config:
           credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials
           enableSharedConfig: "true"
           region: ${REGION}
         provider: aws
```

Comment (weshayutin) [reply content/docs/misc/oadp/rosa-sts/_index.md:229]: @shawn-hurley happy to work w/ you on snapLocation, didn't work for me.

**Comment by weshayutin** [comment]
@jewzaam need some mobb doc reviews, do you know who can help?

**Comment by jewzaam** [comment]
@weshayutin is there a JIRA associated with this?  Probably something PM can help wrangle but hoping JIRA would have some clues for who needs to be involved.

**Comment by weshayutin** [comment]
@paulczar @scottd018 @cwooley-rh @dsari-rh looking for reviews re: the OADP mobb doc for rosa sts.
Thanks in advance!

**Comment by cwooley-rh** [comment]
reviewing this rn

**Review by kaovilai — APPROVED** [review]

**Review by cwooley-rh — APPROVED** [review]
tested, looks good

**Pull request merged**